deploy/examples/create-external-cluster-resources.py

```diff
@@ -1821,6 +1913,35 @@ def gen_json_out(self):
                     }
                 )
 
+        # if 'TOPOLOGY_POOLS', 'TOPOLOGY_FAILURE_DOMAIN_LABEL', 'TOPOLOGY_FAILURE_DOMAIN_VALUES'  exists,
+        # then only add 'topology' StorageClass
+        if (
+            self.out_map["TOPOLOGY_POOLS"]
+            and self.out_map["TOPOLOGY_FAILURE_DOMAIN_LABEL"]
+            and self.out_map["TOPOLOGY_FAILURE_DOMAIN_VALUES"]
+        ):
+            json_out.append(
+                {
+                    "name": "ceph-rbd-topology",
+                    "kind": "StorageClass",
+                    "data": {
+                        "topologyFailureDomainLabel": self.out_map[
+                            "TOPOLOGY_FAILURE_DOMAIN_LABEL"
+                        ],
+                        "topologyFailureDomainValues": self.convert_comma_seprated_to_array(
```

Comment:
as per the request from the ocs operator https://github.com/red-hat-storage/ocs-operator/pull/2501#discussion_r1522623358 we need to re look to the formatting, so I would do that in follow up PR if agreed

Reply:
What is the suggestion for formatting? Or it's not clear yet?

Reply:
updated, we can also add spell check in python script later

Reply:
@travisn What's your opinion on this?

Reply:
Final discussion we decided to keep in ocs operator